BenmoussaYounes/chat-app
# 5-setup firebase and link it to the project ✨

## JiraTicket [FI-5](https://contributors.atlassian.net/jira/software/projects/ATX/boards/6?selectedIssue=ATX-5)

## Description 📑
- link project with firebase to be able to use firestore service

## Type of Change


## Changes

This pull request includes the following changes:

- setup firebase for android app both dev and production flavors
- setup firebase for ios app
- add cloud_firestore  package to enable interaction with firestore
- refactor android/ios app id from com.attraxia.chat_app to com.attraxia.chatapp since it was not a valid bundle id for firebase

## How to test 🚦
- build the app with both development and production flavors and check if it running correctly
